.github/workflows/toolchain-version-check.yaml

```diff
@@ -0,0 +1,76 @@
+---
+name: Toolchain Version Check
+"on":
+  schedule:
+    - cron: "0 5 * * 5" # 5am UTC Fridays
```

Comment:
can we add a note that we do this on Fridays because new Rust versions get released on Thursdays?